tools/InterfaceGenerator/generator/generators/SmartFactoryJSONRPC.py

```diff
@@ -88,9 +88,9 @@ def _gen_schema_params_fill(self, message_type_name):
         return u"".join(
             [self._base_params,
              self._correlation_id_param
-                if unicode(message_type_name) != u"notification" else u"",
```

Comment:
Is the removal of the unicode method related to the change from python2 to python3?

Reply:
yes, python2 has really terrible support of string encoding thats why there was even separate function - "unicode", however python3 has really good supporting of string encoding, so it doesn't have function - "unicode" at all.